core/src/main/java/com/radixdlt/rev2/REv2StateComputer.java

```diff
@@ -164,7 +164,7 @@ public void addToMempool(MempoolAdd mempoolAdd, NodeId origin) {
                 mempoolAddSuccessEventDispatcher.dispatch(success);
               } catch (MempoolFullException | MempoolDuplicateException ignored) {
```

Comment:
I know this is unrelated to your change, but confused me a little, maybe a `// Ignored` in this empty bracket might make this clearer :)